installer/dev/packages.h

```diff
@@ -11,10 +11,17 @@ using namespace Windows::System;
 
 namespace ProjectReunionInstaller {
 
+    enum class DeploymentBehavior
+    {
+        Default,        // Package will be deployed based on its properties alone.
+        Framework       // Package will be treated as a framework, even if it is not.
```

Comment:
NIT: `Framework,`

Last symbol in the enum can have a trailing comma (not an error, for 5-10 years now) and this way future additions make for simpler diffs (this line won't change because you don't need to add a comma)